en/deploy-on-aws-eks.md

```diff
@@ -300,485 +133,301 @@ You can use the `terraform output` command to get the output again.
     tidb-monitor-6fbcc68669-dsjlc     3/3     Running   0          47h
     tidb-pd-0                         1/1     Running   0          47h
     tidb-pd-1                         1/1     Running   0          46h
+    tidb-pd-2                         1/1     Running   0          46h
     tidb-tidb-0                       2/2     Running   0          47h
     tidb-tidb-1                       2/2     Running   0          46h
     tidb-tikv-0                       1/1     Running   0          47h
     tidb-tikv-1                       1/1     Running   0          47h
     tidb-tikv-2                       1/1     Running   0          47h
-    tidb-tikv-3                       1/1     Running   0          46h
     ```
 
-### Enable cross-zone load balancing for the LoadBalancer of the TiDB service
-
-Due to an [issue](https://github.com/kubernetes/kubernetes/issues/82595) of AWS Network Load Balancer (NLB), the NLB created for the TiDB service cannot automatically enable cross-zone load balancing. You can manually enable it by taking the following steps:
-
-1. Get the name of the NLB created for the TiDB service:
-
-    {{< copyable "shell-regular" >}}
-
-    ```shell
-    kubectl --kubeconfig credentials/kubeconfig_${eks_name} get svc ${default_cluster_name}-tidb -n ${namespace}
-    ```
+## Access the database
 
-    This is an example of the output:
+After you deploy a TiDB cluster, you can access the TiDB database via MySQL client.
 
-    ```
-    kubectl --kubeconfig credentials/kubeconfig_test get svc test-tidb -n test
-    NAME        TYPE           CLUSTER-IP      EXTERNAL-IP                                                                     PORT(S)                          AGE
-    tidb-tidb   LoadBalancer   172.20.39.180   a7aa544c49f914930b3b0532022e7d3c-83c0c97d8b659075.elb.us-west-2.amazonaws.com   4000:32387/TCP,10080:31486/TCP   3m46s
-    ```
+### Prepare a machine that can access the cluster
```

Comment:
Change the `machine` in this file to `host`?
`Bastion host` is used in the official docs https://aws.amazon.com/blogs/security/tag/bastion-host/